src/squlearn/util/decompose_to_std.py

```diff
@@ -0,0 +1,15 @@
+from qiskit.circuit.library import standard_gates
+
+def DecomposeToStd(circuit):
+        """Function to decompose the circuit to standard gates"""
+        decompose_names = [None]
+        while len(decompose_names)>0:
+            decompose_names = []
+            for instr in circuit.data:
```

Comment:
```suggestion
            for instruction in circuit.data:
```
No need to use abbreviations here.